packages/frontend/components/Forms/FormTabs.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import styles from './Form.module.scss'
+
+export default function FormTabs({ onChange }) {
+  const [currentTab, setCurrentTab] = useState('message')
```

Comment:
could make the tabs use an enum to prevent spelling mistakes but this is so minor that ill leave it up to you if you want to change it.

Reply:
Yeah, thought about it, but forgot to implement it. I'll make this

Reply:
Done